src/components/Characters.css

```diff
@@ -0,0 +1,37 @@
+.MoreButton {
+  padding: 2%;
+  background-color: orange;
+  border-radius: 4px 4px 4px 4px;
```

Comment:
you can just say 4px on its own as that will cover all four sides :)

Reply:
👍